### PR TITLE
Implement timeout.

### PR DESCRIPTION
**Pull request by 7415963987456321**
This commit implements the timeout feature such that a user can set the timeout of wob with a command line argument '-t [milliseconds]'.  The arguments are parsed with a standard getopt loop, this comes in handy for future additions.

### CODE REVIEW

**Comment by francma** [comment]
I think it would be great to have it accept milliseconds instead of seconds. This way user can define intervals shorter than 1 sec :)

And also please try to make the code look like the rest of the file - space before opening `(` and so on - thanks.

**Comment by 7415963987456321** [comment]
@francma Yeah, it makes more sense to have it accept milliseconds. I have addressed that as well as the style guide.

File: wob.c [hunk 1]
```diff
@@ -274,10 +274,27 @@ main(int argc, char **argv)
...
+				timeout_msec = atoi(optarg);
+				if (timeout_msec < 0){
+					fprintf(stderr, "Timeout must be a positive value.");
+					return EXIT_FAILURE;
+				}
+				break;
+		}
+	}
+
+	uint32_t * argb = wob_create_argb_buffer(&app), i, k;
```
Comment (7415963987456321) [inline wob.c]: This got added while resolving merge conflicts. No worries, it's supposed to be there anyway,

**Comment by 7415963987456321** [comment]
Eh something funky going on while resolving the merge conflicts, It should be fine now, except I keep getting that one commit that is 'Merge to master', Everything is correct though.

**Comment by francma** [comment]
Sorry for the delay :slightly_frowning_face: 

I'm currently working on fixing https://github.com/francma/wob/issues/3 and this kind of conflicts with my changes (that `struct timeval timeout` part, `getopt` is fine). I'll get back to you once I'm done. Thanks :slightly_smiling_face:

**Comment by 7415963987456321** [comment]
Alright, do you mind if I expand on this a little bit. I have a branch implementing a version flag, and I've finished the maximum feature. I might just push all those features into this branch if you don't mind.

**Comment by francma** [comment]
Create 1 PR per 1 feature, preferably.

**Pull request merged**